Makefile

```diff
@@ -36,12 +33,26 @@ build: mod
 	@go build $(BUILD_FLAGS) -o build/ ./cmd/celestia-appd
 .PHONY: build
 
+## build-multiplexer: Builds with the "multiplexer" build tags.
+build-multiplexer: mod
```

Comment:
I don't know if we want the default to actually be the multiplexer. Users shouldn't need to know what a multiplexer is - they should just build the binary as they did normally at it have the same behaviour as before i.e. single binary syncs